org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/RuleEngineParsingTest.xtend

```diff
@@ -0,0 +1,29 @@
+/*
+ * generated by Xtext */
+package org.eclipse.xtext.example.homeautomation.tests
+
+import com.google.inject.Inject
+import org.eclipse.xtext.example.homeautomation.ruleEngine.Model
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.XtextRunner
+import org.eclipse.xtext.testing.util.ParseHelper
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(XtextRunner)
+@InjectWith(RuleEngineInjectorProvider)
+class RuleEngineParsingTest{
+
+	@Inject
+	ParseHelper<Model> parseHelper
+
+	@Test 
+	def void loadModel() {
+		val result = parseHelper.parse('''
+			Hello Xtext!
```

Comment:
this test would fail, wouldn't it? "Hello Xtext!" does not seem to valid syntax of the rule engine lang.